code/modules/mob/living/simple_animal/abnormality/waw/warden.dm

```diff
@@ -0,0 +1,95 @@
+/mob/living/simple_animal/hostile/abnormality/warden
+	name = "The Warden"
+	desc = "An abnormality that takes the form of a fleshy stick wearing a dress and eyes. You don't want to know what's under that dress."
+	icon = 'ModularTegustation/Teguicons/48x64.dmi'
+	icon_state = "warden"
+	icon_living = "warden"
+	maxHealth = 1700
+	health = 1700
+	pixel_x = -8
+	base_pixel_x = -8
+	damage_coeff = list(BRUTE = 0.7, RED_DAMAGE = 0.7, WHITE_DAMAGE = 1.2, BLACK_DAMAGE = 0.4, PALE_DAMAGE = 1.2)
```

Comment:
I recommend making pale resistance higher, i.e. 0.6, since 1.2 effectively means you can kill it in 5 hits with Justitia.